dwave/embedding/transforms.py

```diff
@@ -92,11 +193,25 @@ def embed_bqm(source_bqm, embedding, target_adjacency, chain_strength=1.0,
         :func:`.embed_ising`, :func:`.embed_qubo`
 
     """
+    if embedded_structure is None:
+        if embedding is None or target_adjacency is None:
+            raise ValueError(("either embedded_structure or (embedding and "
+                              "target_adjacency) should be provided, but not both"))
+        edgelist = adjacency_to_edgelist(target_adjacency)
+        embedded_structure = EmbeddedStructure(edgelist, embedding)
+    elif embedding is not None or target_adjacency is not None:
+#        raise ValueError(("either embedded_structure or (embedding and "
```

Comment:
I think you meant to leave this in.

Reply:
I meant to entice you to suggesting a better path forward :stuck_out_tongue_closed_eyes:.  Success!